test/local/account_routes.js

```diff
@@ -90,17 +91,19 @@ test('/recovery_email/status', function (t) {
   })
   var route = getRoute(accountRoutes, '/recovery_email/status')
 
-  test('sign-in confirmation disabled', function (t) {
+  t.test('sign-in confirmation disabled', function (t) {
+    t.plan(2)
     config.signinConfirmation.enabled = false
 
-    test('invalid email', function (t) {
+    t.test('invalid email', function (t) {
```

Comment:
this seems to be the better way to nest tests to avoid issues with leftover tests, 

ref: https://ci.testling.com/guide/tape#nesting-tests